src/components/FilmList/Films.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import './Films.scss';
+import { isEmptyObject, isNonEmptyArray } from './deps';
+import { ListElement } from './type';
+
+class Films extends React.Component<ListElement> {

```

Comment:
Why use the plural "Films"?
This is a detail view, so it should have a singular name, like "Film" or "FilmDetail".

Reply:
ok